src/pow.cpp

```diff
@@ -15,9 +15,15 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
     assert(pindexLast != nullptr);
     unsigned int nProofOfWorkLimit = UintToArith256(params.powLimit).GetCompact();
 
-    // Only change once per difficulty adjustment interval
```

Comment:
did you need me to fill this in? 
not sure if missing or if params call for this syntax

Reply:
Sorry, I didn't get it. I basically added a condition to handle premine period. That's saying, the difficulty will be forced to lower bound if the height of the next block is between [BTGHeight, BTGHeight + BTGPremineWindow].

Reply:
Whats this premine for?

Reply:
PREMINE SUCKS

Reply:
This is a pretty large premine, reddit is in an uproar and calling this project a scam because of this.  I would think twice on this guys.

Reply:
y (price/trust/investor/user confidence in BTG @ fork date 25/10/17): f(x) = 1/x (unilaterally/unexplained self-initiated super large 16,000 block * 12.5 = 200,000 BTG (@? price) premine to ?supposedly ?fund developers) == too greddy. TLDR ++ premine ++ undermine (confidence)

Reply:
Which of devs thought a premine was a good idea?

Reply:
Hello guys! Please check this pull request - https://github.com/h4x3rotab/BTCGPU/pull/1/files . This code for the premining has been made a month ago and a week after the concept was changed. Please have in mind that a premining is being discussed and if there is one, it will be completely transparent to everyone.

Reply:
@StarbuckBG Okay, how about transparency on exactly which block the fork will occur? Or did that happen already?

Reply:
i would like to give about 150kzols hashing power. how can i support your directly during the pre-mine stage?